example-project/plugins/dummy_plugin/src/plugin.ts

```diff
@@ -36,6 +37,9 @@ interface WorkflwoPayloadDeserialized {
   count: number;
 }
 
+const randomInt = (min: number, max: number) =>
```

Comment:
Inspired https://github.com/wormhole-foundation/relayer-engine/issues/30#issue-1521214247

Reply:
Agreed.

Reply:
should I do that in this PR or do you want to tackle separatedly?

Reply:
One of us should do it separately 👍